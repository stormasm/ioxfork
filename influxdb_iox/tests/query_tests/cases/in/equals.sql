-- Basic query tests for measurement names that have equals in their names
-- IOX_SETUP: EqualInMeasurements

-- query data
SELECT * from "measurement=one";



-- projection
SELECT tag from "measurement=one";
