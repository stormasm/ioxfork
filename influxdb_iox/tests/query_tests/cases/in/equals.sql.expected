-- Test Setup: EqualInMeasurements
-- SQL: SELECT * from "measurement=one";
+-------+--------+--------------------------------+
| field | tag    | time                           |
+-------+--------+--------------------------------+
| 1.0   | value  | 2021-01-01T00:00:01.000000001Z |
| 1.0   | value2 | 2021-01-01T00:00:01.000000002Z |
+-------+--------+--------------------------------+
-- SQL: SELECT tag from "measurement=one";
+--------+
| tag    |
+--------+
| value  |
| value2 |
+--------+